client/src/components/DataFiles/DataFilesSystemSelector/DataFilesSystemSelector.js

```diff
@@ -105,13 +108,15 @@ DataFilesSystemSelector.propTypes = {
   section: PropTypes.string.isRequired,
   disabled: PropTypes.bool,
   operation: PropTypes.string.isRequired,
-  showProjects: PropTypes.bool
+  showProjects: PropTypes.bool,
+  excludedSystems: PropTypes.arrayOf(PropTypes.string)
 };
 
 DataFilesSystemSelector.defaultProps = {
   systemId: '',
   disabled: false,
-  showProjects: false
+  showProjects: false,
+  excludedSystems: []
```

Comment:
Can you leave a comment here for the string for excludedSystems? Is it the configuration system name or hostname of the actual system?

Reply:
I believe it's the `system` field of `_PORTAL_DATAFILES_STORAGE_SYSTEMS` or local systems in the response from `SystemListingView`:
```
    {
        'name': 'Google Drive',
        'system': 'googledrive',
        'scheme': 'private',
        'api': 'googledrive',
        'icon': None
    }
```